DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/_CourseDetailsExpandable.cshtml

```diff
@@ -0,0 +1,72 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@model CourseDetailsViewModel
+
+<details class="nhsuk-details nhsuk-expander">
+  <summary class="nhsuk-details__summary">
+    <span class="nhsuk-details__summary-text">
+      Course details
+    </span>
+  </summary>
+  <div class="nhsuk-details__text">
+    <dl class="nhsuk-summary-list details-list-with-button">
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        @if (string.IsNullOrWhiteSpace(Model.Password)) {
+          <dt class="nhsuk-summary-list__key">
+            Requires password
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            No
+          </dd>
+        } else {
+          <dt class="nhsuk-summary-list__key">
+            Password
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.Password" />

```

Comment:
I don't think we need to use the partial here since there is already a condition around the Password being null or white space.